lib/storage/src/content_manager/consensus_state.rs

```diff
@@ -276,7 +305,10 @@ impl<C: CollectionContainer> ConsensusState<C> {
                     })?,
                 )
                 .map(|()| true),
-            ConsensusOperations::RemovePeer(peer_id) => self.remove_peer(peer_id).map(|()| true),
+            ConsensusOperations::RemovePeer(_peer_id) => {
+                debug_assert!(false, "Do not expect RemovePeer to be directly proposed");
```

Comment:
Is it possible to add more description why it is not expected at this point? It is getting a bit complex.